app/src/main/java/com/google/droidjump/GameView.java

```diff
@@ -45,6 +50,8 @@ public GameView(Context context) {
 
     public GameView(Context context, int screenX, int screenY, boolean isPlaying) {
         super(context);
+        levelData = new LevelData(Level.LEVEL1, getResources());
+        intervalTimePoint = GameConstants.NEW_INTERVAL_TIME_POINT;
         receiveLevelDetails();
         timePoint = 0;
```

Comment:
nit: we can have timePoint= GameConstants.NEW_INTERVAL_TIME_POINT;

Reply:
Done